pages/linux/rusnapshot.md

```diff
@@ -0,0 +1,20 @@
+# rusnapshot
+
+> BTRFS snapshotting utility made in Rust.
+> More information: <https://github.com/Edu4rdSHL/rusnapshot>.
+
+- Create a snapshot using a config file:
+
+`sudo rusnapshot --config {{/path/to/config.toml}} --cr`
+
+- List created snapshots:
+
+`sudo rusnapshot -c {{/path/to/config.toml}} -l`
+
+- Delete a snapshot by ID or the name of the snapshot:
+
+`sudo rusnapshot -c {{/path/to/config.toml}} --del --id {{snapshot_id}}`
+
+- Delete all snapshots of [kind] three-hours:
+
+`sudo rusnapshot -c {{/path/to/config.toml}}  -l -k 0 --clean --kind three-hours`
```

Comment:
```suggestion
`sudo rusnapshot --config {{/path/to/config.toml}}  --list --keep 0 --clean --kind {{three-hours}}`
```

Reply:
I think I made most of the changes 
I'll add the brackets for the variable

Reply:
Please take a look again if you find something I'll edit it right at it but I think for the most part is fine.